fortify-issue-manager/src/main/java/com/microfocus/security/automation/fortify/issue/manager/BugTrackerSettings.java

```diff
@@ -17,14 +17,18 @@
 
 import java.util.Map;
 
-public final class BugTrackerSettings
+public class BugTrackerSettings
```

Comment:
BugTrackerSettings, OctaneBugTrackerSettings, OctaneLoginException could be moved to the tracker package.

Reply:
Done